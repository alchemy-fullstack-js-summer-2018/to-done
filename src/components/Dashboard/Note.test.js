// import NoteForm from './NoteForm';
// import React from 'react';
// import { mount } from 'enzyme';
// import toJSON from 'enzyme-to-json';

describe('Note', () => {

  it('testing...', () => {

  });
});