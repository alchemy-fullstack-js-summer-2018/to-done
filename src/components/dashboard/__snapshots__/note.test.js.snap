// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note render display or edit 1`] = `
<li>
  <NoteDisplay
    note={
      Object {
        "content": "Sunday, shower day :)",
        "key": "123",
        "title": "shower",
      }
    }
    onDelete={[Function]}
    onEdit={[Function]}
  />
</li>
`;

exports[`Note render display or edit 2`] = `
<li>
  <NoteDisplay
    note={
      Object {
        "content": "Sunday, shower day :)",
        "key": "123",
        "title": "shower",
      }
    }
    onDelete={[Function]}
    onEdit={[Function]}
  />
</li>
`;
